src/main/scala/com/mwz/sonar/scala/util/package.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Sonar Scala Plugin
+ * Copyright (C) 2018 All contributors
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package com.mwz.sonar.scala
+
+import java.util.Optional
+import scala.language.implicitConversions
+
+/**
+ *  Util methods and conversions
+ *
+ *  @note The Scala.Option <-> Java.Optional conversions were taken from
+ *  https://gist.github.com/julienroubieu/fbb7e1467ab44203a09f
```

Comment:
Looks like the [scala-java8-compat](https://github.com/scala/scala-java8-compat#converters-between-scalaoption-and-javautil-classes-optional-optionaldouble-optionalint-and-optionallong) library would have been useful in this case, but yeah I'm happy to keep this if we just need `Option` conversions.